### PR TITLE
Bug 1570174 stop serving custom font locale css

### PR DESCRIPTION
**Pull request by schalkneethling**
This removes the few locale-specific style sheets we were loading to specify some custom fonts for body copy, and in some cases headings. It also inlines the CSS used for locales that are well supported by ZillaSlab.

I used [this page](http://clagnut.com/blog/2380/) to test Arial against the various languages and also tested the list of locales for which we were loading custom fonts locally.

I did not find any instances where characters were not supported. @peterbe r?

### CODE REVIEW

**Comment by schalkneethling** [comment]
@peterbe updated

File: jinja2/includes/zilla.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+    @font-face {
+        font-family: zillaslab;
+        font-display: 'swap';
+        src: url({{ static('fonts/locales/ZillaSlab-Bold.subset.woff2') }}) format('woff2'),
+             url({{ static('fonts/locales/ZillaSlab-Bold.subset.woff') }}) format('woff');
+        font-weight: bold;
+        font-style: normal;
+    }
+  </style>
```
Comment (peterbe) [inline jinja2/includes/zilla.html]:
```suggestion
</style>

```

File: jinja2/includes/zilla.html [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<style>
+    @font-face {
+        font-family: zillaslab;
+        font-display: 'swap';
```
Comment (peterbe) [inline jinja2/includes/zilla.html]:
```suggestion
        font-display: swap;
```

File: jinja2/includes/zilla.html [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        src: url({{ static('fonts/locales/ZillaSlab-Regular.subset.woff2') }}) format('woff2'),
+             url({{ static('fonts/locales/ZillaSlab-Regular.subset.woff') }}) format('woff');
+        font-weight: normal;
+        font-style: normal;
+    }
+
+    @font-face {
+        font-family: zillaslab;
+        font-display: 'swap';
```
Comment (peterbe) [inline jinja2/includes/zilla.html]:
```suggestion
        font-display: swap;
```

**Pull request merged**